app/api/migrations/0009_tag.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2 on 2021-04-13 16:50
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0008_auto_20210302_1013'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Tag',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('text', models.TextField()),
+                ('project', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='tags', to='api.project')),
+            ],
```

Comment:
The fields look good but I guess are not enough as per the request spec,the required fields are

> 1. id (PK)
> 2. text
> 3. created_at
> 4. project_id (FK)
> 5. text_color
> 6. background_color

as per the spec here: https://github.com/doccano/doccano/issues/1044

If you have another idea for text and background color please let me know. But I believe created_at/create_dt_tm is an important field to have.

Reply:
What would be the purpose of a creation date?

Reply:
I believe it is a good idea to log when a resource is created or deleted in general. Today the purpose of the tags are limited but in the future, we could add a page(view) showing all projects for a specific tag, and on that page, we might want to sort the projects by most-to-least (or vice versa) recently tagged projects. This might be helpful for a user to figure out the priority based on the date last tagged. 

Do let me know your thoughts if you feel the above-mentioned are not valid use cases or are covered as part o other issues.

Reply:
The additional of a creation date is not currently needed and will be of limited use in the future.
The priority of a project can also be indicated by tags instead of your most recently tagged project criterion.
After all one might expand the tag functionality by color or created_at in the future if needed.

Reply:
Well, I agree that it is of limited use and you do have a point about priority being represented as a tag, did not think of it that way. Sounds good to me!